### ISSUE TITLE
Integration with the new v10 journals page system

### ISSUE THREAD

**Issue by lucaspicerni** [zeel01/poi-teleport#15]
I create a journal with locations, and distribute images and descriptions of these locations between pages. Many of the pages are set to notes for a scene, some are not.

If I create a journal pin from a page that is set to a scene note, everything works normally. However, if I create a journal pin from a page that is not defined as a scene note, but is in the same journal with other pages that are scene notes, then the pin still works as a POI teleporter anyway. It redirects to the first page in the journal that links to a scene.

I believe that, with the new format of journal pages, the POI Teleporter should take into account the page linked to the scene, and not the journal as a whole.

**Comment by zeel01** [comment]
I agree, this is a good enhancement. I can't really give any guarantee of how soon you would see it though. I've unfortunately not had a lot of time to work on this recently.

**Comment by tmtProdigy** [comment]
Just want to weigh in and let you know i would really appreciate this too, happy to chip in with a couople bucks in case you have a "buy me a coffee" or something. cheers
